src/ui/server/test-mocks/index.ts

```diff
@@ -22,7 +22,9 @@ const mockReq = () => {
     },
     method: 'GET',
     originalUrl: 'mock',
-    params: {},
+    params: {
+      referenceNumber: '12345',
```

Comment:
Nice - this'll save as defining in the unit tests, can improve those after this PR 🚀 

Could we pull this in from mockApplication?